pkg/controller/vsphere/util_test.go

```diff
@@ -15,19 +15,22 @@ const (
 	testRegion       = "testRegion"
 	testZone         = "testZone"
 	testPort         = "443"
-	testInsecureFlag = "1"
+	testInsecureFlag = true
```

Comment:
Is there a change in the accepted values?

Reply:
no, its a boolean that comes out of the object, not a "1".  below you can see "1" is still used in creating ini file.